### ISSUE TITLE
validation problems for fields inside a fieldGroup

### ISSUE THREAD

**Issue by pselis** [ngx-formly/ngx-formly#1677]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
Using  formly inside a mat-dialog, when I open the dialog for the second time, the 
validation do not work for the fields inside a fieldGroup.

**Minimal reproduction of the problem with instructions**
https://stackblitz.com/edit/angular-3rd8sk-exygim

1) click on open button
2) click on submit -> the form is invalid
3) close the dialog
4) click on open button
5) click on submit -> the form is valid

this happens only for the fields inside a fieldGroup

Thanks for all

**Comment by aitboudad** [comment]
using `immutable` would fix the issue, you just need to pass it through the `extras` option:

```patch
-- FormlyModule.forRoot({ immutable: true }),
++ FormlyModule.forRoot({ extras: { immutable: true } }),
```

**Comment by aitboudad** [comment]
a fix is landed in `5.3` version.
